RecoEgamma/EgammaPhotonProducers/python/reducedEgamma_cfi.py

```diff
@@ -104,3 +104,14 @@
     hiPhotonIsolationMapInput = "photonIsolationHIProducerppGED",
     hiPhotonIsolationMapOutput = "photonIsolationHIProducerppGED"
 )
+
+from RecoHI.HiJetAlgos.HiBadParticleCleaner_cfi import cleanedParticleFlow
+from CommonTools.ParticleFlow.pfEGammaToCandidateRemapper_cfi import pfEGammaToCandidateRemapper as pfEGammaToCandidateRemapperCleaned
+pfEGammaToCandidateRemapperCleaned.pf2pf = cms.InputTag("cleanedParticleFlow")
```

Comment:
Please clone `pfEGammaToCandidateRemapperCleaned` from `pfEGammaToCandidateRemapper` inside the pfEGammaToCandidateRemapper_cfi.py: modules should be modified in the very same configuration file in which they are defined, not in random places elsewhere in cmssw.
(You can follow as example how the `primaryVertexAssociationCleaned` is cloned and modifiel inside primaryVertexAssociation_cfi.py)